resources/js/components/Tool.vue

```diff
@@ -32,12 +33,40 @@
                         <td>{{ job.withoutOverlapping ? 'Yes' : 'No' }}</td>
                         <td>{{ job.onOneServer ? 'Yes' : 'No' }}</td>
                         <td>{{ job.evenInMaintenanceMode ? 'Yes' : 'No' }}</td>
+                        <td>
+                            <button                                
+                                title="Dispatch"
+                                class="appearance-none mr-3"
+                                :class="canDispatchCommand(job.command) ? 'text-70 hover:text-primary' : 'cursor-default text-40'"
+                                :disabled="!canDispatchCommand(job.command)"
+                                @click.prevent="openConfirmationModal(job)"
+                            >
+                                <icon type="play" />
+                            </button>
+                        </td>
                     </tr>
                 </tbody>
             </table>
-
+            <portal to="modals">
+                <transition name="fade">
+                    <delete-resource-modal
```

Comment:
@tanjemark Could we use the `ConfirmActionModal.vue` component here instead? Seems like it would be a better fit. Also not sure if it useable for this, was browsing the prebult modal's in Nova and that just seemed like the best fit name wise.